modules/installation-bootstrap-gather.adoc

```diff
@@ -17,13 +18,9 @@ running cluster, use the `oc adm must-gather` command.
 
 .Prerequisites
 
-* Your {product-title} installation failed before the bootstrap process
-finished. The bootstrap node must be running and accessible through SSH.
-* The `ssh-agent` process is active on your computer, and you provided both the
-`ssh-agent` process and the installation program the same SSH key.
-* If you tried to install a cluster on infrastructure that you provisioned, you
-must have the fully-qualified domain names of the control plane, or master,
-machines.
+* Your {product-title} installation failed before the bootstrap process finished. The bootstrap node is running and accessible through SSH.
+* The `ssh-agent` process is active on your computer, and you provided both the `ssh-agent` process and the installation program the same SSH key.
```

Comment:
Maybe "and you provided the same SSH key to both the `ssh-agent` process and the installation program" would sound better?

Reply:
Resolved.